plugins/pci-psp/meson.build

```diff
@@ -13,4 +13,31 @@ plugin_builtins += static_library('fu_plugin_pci_psp',
   c_args: cargs,
   dependencies: plugin_deps,
 )
+if intospection_integration_tests
+  envs = environment()
+  envs.set('DAEMON_BUILDDIR', join_paths(meson.project_build_root(), 'src'))
+  envs.set('LIBFWUPD_BUILD_DIR', join_paths(meson.project_build_root(), 'libfwupd'))
+  envs.set('PYTHONPATH', join_paths(meson.project_source_root(), 'src'))
+  envs.set('FWUPD_DATADIR_QUIRKS', join_paths(meson.project_build_root()))
+  envs.set('STATE_DIRECTORY', join_paths(meson.project_build_root(), 'state'))
```

Comment:
given we're likely to have a few of these, how about a `env_unittest_inspector` defined in `plugins/meson.build` -- or something like that?